collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -186,8 +186,8 @@ void addSample(MetricFamilySamples.Sample sample, Type type, String help) {
 
       private void defaultExport(
           String domain,
-          LinkedHashMap<String, String> beanProperties,
```

Comment:
That this has a specific order is an important part of the interface, so these should stay as is.